src/Blocks/custom/form/components/form-custom-options.js

```diff
@@ -1,48 +1,41 @@
 import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 import { TextControl } from '@wordpress/components';
-
+import manifest from '../manifest.json';
+import { getAttrKey } from '@eightshift/frontend-libs/scripts/helpers';
 
 /**
  * Options component
  *
  * @param {object} props Component props.
  */
-export const FormCustomOptions = (props) => {
-  const {
-    action,
-    method,
-    target,
-    onChangeAction,
-    onChangeMethod,
-    onChangeTarget,
-  } = props;
+export const FormCustomOptions = (attributes) => {
+	const {
+		setAttributes,
+		formAction,
+		formMethod,
+		formTarget,
+	} = attributes;
 
-  return (
-    <Fragment>
-      {onChangeAction &&
-        <TextControl
-          label={__('Action', 'eightshift-forms')}
-          value={action}
-          onChange={onChangeAction}
-        />
-      }
+	return (
+		<Fragment>
+			<TextControl
+				label={__('Action', 'eightshift-forms')}
+				value={formAction}
+				onChange={(value) => setAttributes({ [getAttrKey('formAction', attributes, manifest)]: value })}
+			/>
 
-      {onChangeMethod &&
-        <TextControl
-          label={__('Method', 'eightshift-forms')}
-          value={method}
-          onChange={onChangeMethod}
-        />
-      }
+			<TextControl
+				label={__('Method', 'eightshift-forms')}
+				value={formMethod}
+				onChange={(value) => setAttributes({ [getAttrKey('formMethod', attributes, manifest)]: value })}
+			/>
 
-      {onChangeTarget &&
-        <TextControl
-          label={__('Target', 'eightshift-forms')}
-          value={target}
-          onChange={onChangeTarget}
-        />
-      }
-    </Fragment>
-  );
+			<TextControl
+				label={__('Target', 'eightshift-forms')}
+				value={formTarget}
+				onChange={(value) => setAttributes({ [getAttrKey('formTarget', attributes, manifest)]: value })}
+			/>
+		</Fragment>
```

Comment:
Why not `<>`?

Reply:
This is relatively old code, didn't refactor everything, just made things work (most of the files are still on spaces). Will fix all spaces / tabs after you review the files so it doesn't cause too much diff now. Will also fix fragments.